date-input.html

```diff
@@ -69,6 +70,7 @@
           readonly$="[[readonly]]">
       <span>/</span>
       <input is="iron-input"
+          id="expYear"
```

Comment:
I would rename this to `dateYear` or `expirationYear`. Abbreviations are icky :)